src/User.php

```diff
@@ -35,20 +35,21 @@ public function getUsername(): string
 
     public function getAvatar(): string
     {
-        return $this->avatar;
+        return "./assets/imgs/avatars/" . $this->avatar;
```

Comment:
it would be better to rename this method to `getFullAvatarPath` because it is no longer a simple getter. then it will also need to be unit tested because it is doing more than just getting